INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+# ToDo App Docker Instructions
+
+## Docker Hub Repository
+- The Docker image is available at: https://hub.docker.com/r/evgeniychernyak/todoapp:1.0.0
+
+## Build and Run the Docker Image
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/evgeniychernyak/devops_todolist.git
+   cd devops_todolist
```

Comment:
The instructions stop after cloning the repository. Consider adding steps to build the Docker image using the Dockerfile and run the container. For example, you can include commands like `docker build -t todoapp .` and `docker run -p 8080:8080 todoapp`.